applications/mne_analyze/libs/anShared/Model/averagingdatamodel.cpp

```diff
@@ -146,3 +146,14 @@ QSharedPointer<FIFFLIB::FiffInfo> AveragingDataModel::getFiffInfo()
     QSharedPointer<FIFFLIB::FiffInfo> pInfo = QSharedPointer<FIFFLIB::FiffInfo>(new FIFFLIB::FiffInfo(m_pFiffEvokedSet->info));
     return pInfo;
 }
+
+//=============================================================================================================
+
+QSharedPointer<FIFFLIB::FiffEvokedSet> AveragingDataModel::getEvokedSet()
+{
+    if (!m_pFiffEvokedSet){
```

Comment:
if this is true it follows that the QSharedPointer is NULL since QSharedPointer is automatically initialized as NULL. So I do not think you need to return a NULL ptr here.